packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -368,11 +379,11 @@ const defaultFormSchema: DefaultPluginsSchemaRecord = reactive({
 // This is specifically used for credential plugins
```

Comment:
Update comment to mention developer plugins too

Reply:
Updated.